examples/medplum-demo-bots/src/health-gorilla/send-to-health-gorilla.ts

```diff
@@ -606,7 +606,7 @@ class HealthGorillaRequestGroupBuilder {
             ...medplumCoverage,
             id: 'coverage' + this.coverages.length,
             beneficiary: createReference(this.patient),
-            payor: undefined,
+            payor: [],
```

Comment:
This technically isn't valid FHIR, since the array is empty the property should be omitted